src/modules/instant-checkout/components/InstantCheckout.vue

```diff
@@ -19,7 +19,7 @@ export default {
   data () {
     return {
       supported: false,
-      country: rootStore.state.checkout.shippingDetails.country ? rootStore.state.checkout.shippingDetails.country : storeView.tax.defaultCountry,
```

Comment:
Any chance to keep the getters (as deprecated) in both modules as for now? To keeep the backward compatibility; 1.11 is too complex to upgrade already and I don’t want to make it even harder; the value of separating these information across different modules isn’t in the end so critical to introduce the breaking changes

Reply:
Yeah it should be easy. The reason why I didn't do this is that changing names is quite easy for someone who want to update to newest version, but you are right it's still introduces new complexity. I'll add deprecates.